micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login/Carousel/Carousel.js

```
@@ -0,0 +1,76 @@
+                        style={{ backgroundImage: `url(${item.image})` }}
+                    >
+                        <div className="carousel-content">
+                            <h2>{t(item.title)}</h2>
+                            <p>{t(item.description)}</p>
+                        </div>
+                    </div>
+                ))}
+
+                {/* Combined Navigation and Indicators (Bottom Center) */}
+                <div className="carousel-controls">
+                    <button className="carousel-nav" onClick={prevSlide}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)